app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -2,6 +2,7 @@ class ManageIQ::Providers::Vmware::CloudManager < ManageIQ::Providers::CloudMana
   require_nested :RefreshParser
   require_nested :RefreshWorker
   require_nested :Refresher
+  require_nested :Template
```

Comment:
That's a great question, I haven't seen the `VirtualTemplate` before.  Let me try to find out.